backend/app/Repositories/UserRepository.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Repositories;
+
+use App\Repositories\Contracts\UserRepository as IUserRepository;
+use App\Entities\User;
+
+class UserRepository implements IUserRepository
```

Comment:
for implementation use Eloquent prefix for repositories, that is EloquentUserRepository implements UserRepository